Gemfile

```diff
@@ -36,10 +36,15 @@ gem 'actionview-component'
 
 gem 'uk_postcode'
 
+# Background processing
+gem 'sidekiq'
+gem 'clockwork'
+
 # Logging
 gem 'lograge'
 gem 'logstash-logger'
 gem 'logstash-event'
+gem 'request_store_rails'
```

Comment:
This is a better way to share vars across various context within a request, compared to Thread.current or request_store. It is needed because LogStashLogger operates outside any of the usual rails contexts, such a request/controller etc.